src/GraphQL.php

```diff
@@ -33,6 +33,7 @@ public function getArticle(string $id): array {
       'slug',
       'createdAt',
       'publishedDate',
+      'publishStatus',
```

Comment:
@rkunjappan I tested the API using the GraphQL playground, and it has this `publishStatus` field available.